hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -318,6 +319,53 @@ def get_top_data(topfile):
     return ret
 
 
+def mask_passwords(unmasked_object):
+
+    masked_object = copy.deepcopy(unmasked_object)
+
+    try:
+        mask_file  = __salt__['cp.cache_file']('salt://mask.yaml')
+        mask = yaml.safe_load(open(mask_file))
```

Comment:
Rather than relying on the try:except:, can we check whether `mask_file` is False here and just return?